internal/view/content.html

```diff
@@ -62,19 +62,18 @@
 					var opts = JSON.parse(localStorage.getItem("shiori-account")) || {},
 						ShowId = (typeof opts.config.ShowId === "boolean") ? opts.config.ShowId : false,
```

Comment:
When we refactor the frontend potentially updating to Vue 3 we need to find a better way to set up the default settings without having them set in multiple places.